internal/pkg/cli/deploy/backend_test.go

```diff
@@ -16,6 +20,40 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func TestBackendSvcDeployer_GenerateCloudFormationTemplate(t *testing.T) {
+	t.Run("ensure resulting CloudFormation template custom resource paths are empty", func(t *testing.T) {
```

Comment:
qq: what about paths like addons path? do we want to keep  them empty as well ?

Reply:
ooh good thinking, let me test that with overrides and see if CDK complains. If it does i'll fix that too